### DISCUSSION TITLE
[BUG] getContainsPlayer attempts to call a nil value, resulting in actors just standing around during action.

### DISCUSSION THREAD

**Discussion by Raetekusu** [ErinaSugino/Starbound-SBR-Mod#37] [Trouble Shooting] [answered]
Basically, as soon as either I or a monster activates a node, both actors will simply stand apart and no action will occur.  However, the Stunned and "Having Sex" timers still tick down in the health/status bar.

The log below is pretty much just me spawning in, attempting to get it to work twice, once via Defeat on a monster and once via Defeat on a spawned NPC, both times failing.

[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/11953138/starbound.log)

**Comment by ErinaSugino** [comment, marked as answer]
Don't assume, just read.
https://github.com/ErinaSugino/Starbound-SBR-Mod/wiki/Advanced-Troubleshooting
